### DISCUSSION TITLE
¿HOW CLEAR ALERTS OR LOGS?

### DISCUSSION THREAD

**Discussion by mahodizae** [Security-Onion-Solutions/securityonion#8055] [General] [answered]
Es posible borrar las alertas registradas en ONION SECURITY?

![ONIONS ALERTS](https://user-images.githubusercontent.com/106709209/171510323-458cc4e3-db34-4f4d-bf10-0f03c9910a94.PNG)

**Comment by dougburks** [comment, marked as answer]
You can click the Acknowledge button as described in the documentation:
https://docs.securityonion.net/en/2.3/alerts.html#data-table
